src/Migrations/Utils/Functions.php

```diff
@@ -178,26 +178,26 @@ public static function generate_html_content(string $classname): array
      * Create a new block.
      *
      * @param string $name - The name of the block.
-     * @param array $attrs - The attributes of the block.
-     * @param array $inner_blocks - The internal blocks.
-     * @param string $inner_html - The internal HTML.
-     * @param array $inner_content - The internal content.
+     * @param array $attrs - The attributes of the block (optional).
+     * @param array $inner_blocks - The internal blocks (optional).
+     * @param string $inner_html - The internal HTML (optional).
+     * @param array $inner_content - The internal content (optional).
      * @return array - The new block.
      */
     public static function create_new_block(
         string $name,
-        array $attrs,
-        array $inner_blocks,
-        string $inner_html,
-        array $inner_content
+        ?array $attrs = [],
```

Comment:
It is not necessary to define as `optional` to parameters that are set by `default`.

```suggestion
        array $attrs = [],
```
The same to the rest of `params`